### PR TITLE
Add 1.30.2 release notes, update the release notes template text

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
`serverless-docs-1.30`+

Issue:
No issue.

Link to docs preview:
Serverless 1.30.2 release notes: https://66939--docspreview.netlify.app/openshift-serverless/latest/about/serverless-release-notes#serverless-rn-1-30-2_serverless-release-notes

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by michaelryanpeter** [comment]
LGTM
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: modules/release-notes-template.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@
 // Versions for the components in New features are here (both for 1.29 and 1.29.1):
 // https://gitlab.cee.redhat.com/serverless/p12n-config/-/blob/release-1.29/config.yaml
 
-{ServerlessProductName} X.YY[.Z] is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+{ServerlessProductName} X.YY[.Z] is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {product-title} are included in the following notes.
```
Comment (mgencur) [inline modules/release-notes-template.adoc:17]: This change renders as `New features, updates, and known issues that pertain to OpenShift Serverless on Red Hat OpenShift Serverless are included in the following notes.` 
I guess it's not correct - repeating "Serverless" twice there.

File: modules/serverless-rn-1-30-2.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-30-2_{context}"]
+= Red Hat {ServerlessProductName} 1.30.2
+
+{ServerlessProductName} 1.30.2 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {product-title} are included in the following notes.
```
Comment (mgencur) [inline modules/serverless-rn-1-30-2.adoc:9]: Same as the previous comment. This problem is repeated many times in this PR.

**Review by mgencur — CHANGES_REQUESTED** [review]
Looks good except for one issue as mentioned below.

File: modules/release-notes-template.adoc (hunk 1 above)
Comment (dsimansk) [reply modules/release-notes-template.adoc:17]: It seems to be related to use of `{product-title}` placeholder. It's the same on public docs as well.

Comment (dsimansk) [reply modules/release-notes-template.adoc:17]: Perhaps product-title used to be a reference to OCP in the previous structure?

Comment (dsimansk) [reply modules/release-notes-template.adoc:17]: `{ocp-product-title}` it's then.

Comment (rh-max) [reply modules/release-notes-template.adoc]: @mgencur @dsimansk Thank you Martin and David. The problem is indeed a recent change of the value of the `product-title` attribute. I've fixed it throughout the docs.

**Comment by dsimansk** [comment]
Thanks, @rh-max!
/lgtm

**Pull request merged**